libnabo_installer.bash

```diff
@@ -0,0 +1,85 @@
+#!/bin/bash
+# =================================================================================================
+#
+# Install libnabo
+#
+# Usage:
+#   $ bash libnabo_installer.bash [<optional flag>]
+#
+# Arguments:
+#   --install-path </dir/abs/path/>     The directory where to install libnabo (absolute path)
+#                                           (default location defined in the .env)
+#   --repository-version 1.0.7         Install libnabo release tag version (default to master branch latest)
```

Comment:
Can we update the tag to the planned release? (1.1.0)

Reply:
@boxanm This is a documentation example on how to use this flag, could be any random number that fit the pattern.